src/ShearStressTransportEquationSystem.C

```diff
@@ -178,21 +189,28 @@ ShearStressTransportEquationSystem::solve_and_update()
     clip_min_distance_to_wall();
     
     // deal with DES option
-    if ( SST_DES == realm_.solutionOptions_->turbulenceModel_ )
+    if ( SST_DES == realm_.solutionOptions_->turbulenceModel_ || ABL_DES_BLEND == realm_.solutionOptions_->turbulenceModel_ )
       sstMaxLengthScaleAlgDriver_->execute();
+      compute_f_ABLBlending();    
```

Comment:
It looks like here you are calling the `compute_f_ABLBlending()` under conditions of both `SST_DES` or `ABL_DES_BLEND`.  It seems like you typically add another `if` statement in other parts of the code, so you can probably just add that here as well.